spec/controllers/news_controller_spec.rb

```diff
@@ -52,9 +54,9 @@
         expect(assigns(:articles)).to_not include(article_unpub)
       end
 
-      it 'should return a list ordered by date and name' do
-        expect(assigns(:articles).to_a).to eq([article_today_c, article_today_a, article_today_b, article_yesterday])
-      end
+     it 'should return a list ordered by date and name' do
+       expect(assigns(:articles).to_a).to eq([article_today_c, article_today_a, article_today_b, article_yesterday])
+     end
```

Comment:
I think the indenting here was originally correct and is now one space too short.